### ISSUE TITLE
Icon Request: Zoom to Selection

### ISSUE THREAD

**Issue by skawaii** [google/material-design-icons#632]
Maybe something like this? I created this for a project and am happy to share (although I make no claims at being a designer).

![image](https://user-images.githubusercontent.com/4439/27750943-10152814-5da8-11e7-9b3a-ac3bb0aa9d94.png)

**Comment by createyourperonalaccount** [comment]
Maybe something like this? I created this for a project and am happy to share (although I make no claims at being a designer, but I tried to follow the MD guidelines).

**Comment by jalpadoshi** [comment]
This icon is now available: (https://fonts.google.com/icons?icon.query=zoom)

**Comment by tphinney** [comment]
Well, there is a "zoom in"—I think that is just a smidge different than "zoom to selection". Note the arrow bit at top left…

**Comment by skawaii** [comment]
> Well, there is a "zoom in"—I think that is just a smidge different than "zoom to selection". Note the arrow bit at top left…

Yup, my original attempt (if i remember correctly; it was nearly five years ago) was to combine a point/arrow with the zoom-in icon.
